### DISCUSSION TITLE
How do I pack a simple skinny jar?

### DISCUSSION THREAD

**Discussion by jennet** [juxt/pack.alpha#107] [Q&A]
I have a clojure project using deps.edn, which was using pack.alpha to build a skinny jar (plus libs dir) in my CI build using `clj -A:pack-staging` and aliases in deps.edn of:
```
    :staging {:extra-paths ["env/staging/resources"]}
    :pack-staging {:extra-deps {pack/pack.alpha {:git/url "https://github.com/juxt/pack.alpha.git" :sha "9fd4a63ddeec3bedcbc819422de5a40ccb9eb8c9"}}
                  :main-opts ["-m mach.pack.alpha.skinny -A:staging"]}
```

This stopped working in the CI, so I've been trying to bring it up to date. I've upgraded clojure tools to 1.11.1.1113, and created a new alias in deps.edn and used the example from your README (adjusted based on issue #97 ) 

```
:pack {:deps {io.github.juxt/pack.alpha {:git/sha "1a077d172afc94e7f7679ecaad57ed9b7dde687d"}}
                  :ns-default juxt.pack.cli.api}
```

It's my understanding (probably wrong!), that running `clojure -T:pack skinny` would build a jar based on the root deps and create a jar a `target` directory? (I've also tried with `-X`) If not, what other values do I need to pass in to the tools command to get it to output files? 
Do you have a hello world example that could be included to show the usage now?

Many thanks 🙇

**Comment by dkarvik** [comment]
You now need to specify either `:libs` and/or `:path`.  What I would guess you're after is this: `clj -T:pack skinny :libs '"target"' :path '"app.jar"' :path-coerce :jar`
